usersync/chooser.go

```diff
@@ -88,6 +84,9 @@ const (
 
 	// StatusBlockedByPrivacy specifies a bidder sync url is not allowed by privacy activities
 	StatusBlockedByPrivacy
+
+	// TODO: Add desciription
+	StatusUnconfiguredBidder
```

Comment:
I just want to understand the motivation for adding this status type. Was this discussed? Is this what PBS-Java does? I see that we are passing all of the known bidders to the chooser which is where we validate the bidders specified in the request and that we are now distinguishing between known bidders that are improperly configured and unknown bidders. That makes sense to me but I wanted to check.

Reply:
Yeah. It's in [the spec](https://github.com/prebid/prebid-server/issues/2173).